packages/selfcare-client-users-updater-consumer/src/config/config.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from "zod";
+import {
+  KafkaConsumerConfig,
+  TokenGenerationConfig,
+} from "pagopa-interop-commons";
+
+export const SelfcareClientUsersUpdaterConsumerConfig = KafkaConsumerConfig.and(
+  TokenGenerationConfig
+).and(
+  z
+    .object({
+      SELFCARE_TOPIC: z.string(),
+      INTEROP_PRODUCT_IDENTIFIER: z.string(),
+      ALLOWED_ORIGINS: z.string(),
+      ALLOWED_ORIGINS_UUID: z.string().uuid(),
+      AUTHORIZATION_PROCESS_URL: z.string(),
+    })
+    .transform((c) => ({
+      selfcareTopic: c.SELFCARE_TOPIC,
```

Comment:
It's the second kind of Selfcare topic in monorepo, in this case I prefer to distinguish it with a clear variable name in the same way reported in Selfcare's documentation : "Topic: sc-users". So I would to rename with:

```suggestion
      selfcareUsersTopic: c.SELFCARE_TOPIC,
```